### PR TITLE
[RFR] isDetailView doesn't work on refernced_list entity id-field

### PR DESCRIPTION
**Pull request by vasiakorobkin**
This PR is absolutely analogous to https://github.com/marmelab/ng-admin/pull/537
This time isDetailLink() applied to `referenced_list` entity doesn't work because of state parameter override.
I've created a e2e-test to demonstrate it. Test fails on your master branch, and passes on my commit.

### CODE REVIEW

File: examples/blog/config.js [hunk 1]
```diff
@@ -133,7 +133,8 @@
                     .targetReferenceField('post_id')
                     .targetFields([
                         nga.field('created_at').label('Posted'),
-                        nga.field('body').label('Comment')
+                        nga.field('body').label('Comment'),
+                        nga.field('id').isDetailLink(true)
```
Comment (jpetitcolas) [inline examples/blog/config.js]: Can you just move the id at the beginning of the table? That's more conventional! :)

**Comment by jpetitcolas** [comment]
Good for me. :+1: 

Can you rebase on master, as tests have been fixed yesterday to ensure everything is correct?

**Comment by jpetitcolas** [comment]
Should fix #569.

**Comment by vasiakorobkin** [comment]
Rebased.

**Comment by jpetitcolas** [comment]
Your PR is still red. I don't think that's an issue with your PR. Looking at it.

**Comment by jpetitcolas** [comment]
Tests are correctly passing on master. Thanks for your help! :)

**Pull request merged**